tests/generators/golden/generated.js

```diff
@@ -427,7 +427,7 @@ function test_constant() {
 
 function mathIsPrime(n) {
   // https://en.wikipedia.org/wiki/Primality_test#Naive_methods
-  if (n === 2 || n === 3) {
```

Comment:
When did the generator test change in this direction?

Reply:
Here: https://github.com/google/blockly/pull/5599/files#diff-7910823a2f8f803e4f1e2aa5e7976728aa490ffe4c015d162e7f2e0533348a1b

This is the one place where functionality inadvertently changed.  But to trigger this change this one has to stuff a string into an untyped variable then pass the variable to the "is prime" math block:
https://blockly-demo.appspot.com/static/demos/code/index.html#384y9p

All the other changes in generated code are verified as being executionally NOPs.